src/mtcp/mtcp_util.c

```diff
@@ -73,6 +73,37 @@ unsigned long mtcp_strtol (char *str)
   return v;
 }
 
+unsigned long mtcp_strtol (const char *str)
+{
+  return (unsigned long) mtcp_strtoll(str);
+}
+
+void mtcp_utoa(char* buffer, unsigned int n)
```

Comment:
How about:
mtcp_utoa -> mtcp_ultoa

'ltoa' is easily recognized as the opposite of 'atol'.  In 'printf' formants, we use %ud to represent an unsigned decimal value.  By analogy, 'ul' should be 'unsigned long'.